libnetwork/osl/namespace_linux.go

```diff
@@ -689,8 +677,18 @@ func setIPv6(path, iface string, enable bool) error {
 func (n *networkNamespace) ApplyOSTweaks(types []SandboxType) {
 	for _, t := range types {
 		switch t {
-		case SandboxTypeLoadBalancer:
-			kernel.ApplyOSTweaks(loadBalancerConfig)
+		case SandboxTypeLoadBalancer, SandboxTypeIngress:
+			kernel.ApplyOSTweaks(map[string]*kernel.OSValue{
+				// disables any special handling on port reuse of existing IPVS connection table entries
+				// more info: https://github.com/torvalds/linux/blob/master/Documentation/networking/ipvs-sysctl.txt#L25:1
```

Comment:
Just a side note: these links are broken, ipvs-sysctl.txt as been converted into rst :wink:

Reply:
Ah, thanks; yes, perhaps better to use a permalink to a specific tag, e.g. https://github.com/torvalds/linux/blob/v5.15/Documentation/networking/ipvs-sysctl.rst#procsysnetipv4vs-variables

(or https://github.com/torvalds/linux/blame/v5.15/Documentation/networking/ipvs-sysctl.rst#L32 to link to the specific option)